lib/rspec/core/formatters/html_formatter.rb

```diff
@@ -69,9 +69,11 @@ def example_failed(failure)
           example = failure.example
 
           exception = failure.exception
+          message_lines = failure.fully_formatted_lines(nil, RSpec::Core::Notifications::NullColorizer)
           exception_details = if exception
                                 {
-                                  :message => failure.message_lines.join("\n"),
+                                  # the regular expression here removes the CLI leading indentation, and any trailing spaces.
+                                  :message => message_lines.drop(2).join("\n").gsub(/^  (.*?)( +)?$/, '\1'),
```

Comment:
I've dropped the identation removal, it only really made sense as a "html won't care about these" kind of step anyway, and to be honest the output looks fine.

I've also added a spec for a multiple line doc string, it has no effect with the current implementation because `message_lines` isn't actual lines, but the array of components, `drop(2)` here is thus completely safe, it wouldn't be if we used `message.lines`